modules/breakpad/SCsub

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
+
+Import("env")
+Import("env_modules")
+
+env_breakpad = env_modules.Clone()
+
+# Thirdparty source files
+
+thirdparty_obj = []
+
+thirdparty_dir = "#thirdparty/breakpad/"
+
+# TODO: find out when these are needed (if at all)
+dwarf_module = False
+stabs_module = False
+
+# Parts of this build script is based on the previous PR trying to implement this
+# https://github.com/godotengine/godot/pull/22778/files
+
+env_breakpad.Append(
+    CPPDEFINES=[
+        "PUBLIC",
+        "_HAS_EXCEPTIONS=0",
+        "_SILENCE_ALL_CXX17_DEPRECATION_WARNINGS",
+    ]
+)
+
+breakpad_src = [
+    "src/common/convert_UTF.cc",
+    # "src/common/long_string_dictionary.cc",
```

Comment:
We tend to remove commented out code that isn't used, unless you want to signal that is is unused as a warning.

Reply:
I'll remove (most) of these commented out things once I'm done. I didn't remove these yet as I'm not sure if these are required for other platforms (I only tested compiling on Linux removing these files from the compile and putting them back in until all linking errors were gone).